.github/workflows/rebaseline-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Rebaseline Tests
+
+on: [workflow_dispatch]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  rebaseline-tests:
+    name: Rebaseline Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+      - name: pip install
+        run: |
+          which python3
+          python3 --version
+          python3 -m pip install -r requirements-dev.txt
+      - name: Install emsdk
+        run: |
+          EM_CONFIG=$HOME/emsdk/.emscripten
```

Comment:
does this need to be `export`ed?

Reply:
The export is below: `echo "EM_CONFIG=$EM_CONFIG" >> $GITHUB_ENV`